### PR TITLE
Irminger 6 deploy and recover updates

### PR DESCRIPTION
**Pull request by rtravis-whoi**
Updated the following with deploy/recover info:
GI01SUMO_Deploy
GI02HYPM_Deploy
GI03FLMA_Deploy
GI03FLMB_Deploy
GI05MOAS-GL525_Deploy
GI05MOAS-GL560_Deploy
GI05MOAS-PG515_Deploy
Also - NUTNRB predeploy cal csvs, and NUTNRM-00760 initial and predeploy cal csvs (still need original vendor cal doc)

### CODE REVIEW

File: deployment/GI03FLMB_Deploy.csv [hunk 1]
```diff
@@ -103,55 +103,55 @@ AR21,,AR30-03,,GI03FLMB-RIS01-00-SIOENG000,4,1,2017-08-07T15:49:00,2018-06-11T19
...
-AR35-05,,,,GI03FLMB-RIM01-02-CTDMOG067,6,1,,,CGMGI-03FLMB-00006,,CGINS-CTDMOG-11603,,,,350,2812,
-AR35-05,,,,GI03FLMB-RIM01-02-CTDMOG068,6,1,,,CGMGI-03FLMB-00006,,CGINS-CTDMOG-12190,,,,500,2812,
-AR35-05,,,,GI03FLMB-RIM01-02-CTDMOH069,6,1,,,CGMGI-03FLMB-00006,,CGINS-CTDMOH-12227,,,,750,2812,
-AR35-05,,,,GI03FLMB-RIM01-02-CTDMOH070,6,1,,,CGMGI-03FLMB-00006,,CGINS-CTDMOH-11703,,,,1000,2812,
-AR35-05,,,,GI03FLMB-RIM01-02-CTDMOH071,6,1,,,CGMGI-03FLMB-00006,,CGINS-CTDMOH-12219,,,,1500,2812,
-AR35-05,,,,GI03FLMB-RIS01-00-SIOENG000,6,1,,,CGMGI-03FLMB-00006,,CGCON-SIOCON-12372,,,,30,2812,
-AR35-05,,,,GI03FLMB-RIS01-03-DOSTAD000,6,1,,,CGMGI-03FLMB-00006,,CGINS-DOSTAD-00224,,,,30,2812,
-AR35-05,,,,GI03FLMB-RIS01-04-PHSENF000,6,1,,,CGMGI-03FLMB-00006,,CGINS-PHSENF-P0183,,,,30,2812,
-AR35-05,,,,GI03FLMB-RIS01-05-FLORTD000,6,1,,,CGMGI-03FLMB-00006,,CGINS-FLORTD-01221,,,,30,2812,
+AR30-03,,AR35-05,,GI03FLMB-RI000-00-CTDMOH000,5,1,2018-06-11T14:35:00,2019-08-10T12:09:00,CGMGI-03FLMB-00005,,CGINS-CTDMOH-10266,59.71427,-39.31329,,1830,2829,
```
Comment (snwhite) [inline deployment/GI03FLMB_Deploy.csv]: All of the OSNAP instruments should be xx29 m

File: deployment/GI02HYPM_Deploy.csv [hunk 2]
```diff
@@ -35,21 +35,21 @@ AR21,,AR30-03,,GI02HYPM-WFP02-04-CTDPFL000,4,1,2017-08-06T13:45:00,2018-06-16T06
...
+AR35-05,,,,GI02HYPM-RIM01-00-SIOENG000,6,1,2019-08-06T12:37:00,,CGMGI-02HYPM-00006,,CGCON-SIOCON-12368,59.97287,-39.53082,,2585,2664,
+AR35-05,,,,GI02HYPM-RIM01-02-CTDMOG039,6,1,2019-08-06T12:37:00,,CGMGI-02HYPM-00006,,CGINS-CTDMOG-11611,59.97287,-39.53082,,138,2664,
+AR35-05,,,,GI02HYPM-WFP02-00-WFPENG000,6,1,2019-08-06T12:37:00,,CGMGI-02HYPM-00006,CGVEH-GWFPGX-77401,CGCON-GWFPGX-77401,59.97287,-39.53082,,2573,2664,
+AR35-05,,,,GI02HYPM-WFP02-01-FLORDL000,6,1,2019-08-06T12:37:00,,CGMGI-02HYPM-00006,CGVEH-GWFPGX-77401,CGINS-FLORDL-02350,59.97287,-39.53082,,2573,2664,
+AR35-05,,,,GI02HYPM-WFP02-03-DOSTAL000,6,1,2019-08-06T12:37:00,,CGMGI-02HYPM-00006,CGVEH-GWFPGX-77401,CGINS-DOSTAL-01106,59.97287,-39.53082,,2573,2664,
+AR35-05,,,,GI02HYPM-WFP02-04-CTDPFL000,6,1,2019-08-06T12:37:00,,CGMGI-02HYPM-00006,CGVEH-GWFPGX-77401,CGINS-CTDPFL-00097,59.97287,-39.53082,,2573,2664,
+AR35-05,,,,GI02HYPM-WFP02-05-VEL3DL000,6,1,2019-08-06T12:37:00,,CGMGI-02HYPM-00006,CGVEH-GWFPGX-77401,CGINS-VEL3DL-01025,59.97287,-39.53082,,2573,2664,
+AR35-05,,,,GI02HYPM-MPM01-02-ZPLSGA009,6,1,2019-08-06T12:37:00,,CGMGI-02HYPM-00006,,CGINS-ZPLSGA-55069,59.97287,-39.53082,,,2664,upward-looking
```
Comment (snwhite) [inline deployment/GI02HYPM_Deploy.csv]: For the newly deployed HYPM-6 at Location A...
Both ZPLSG instruments should have a deployment depth of 150 m.
WFP instruments should have a deployment depth of 2560 m.
The CTDMO should have a deployment depth of 160 m.
The SIOCON should have a deployment depth of 2570 m.

File: deployment/GI02HYPM_Deploy.csv [hunk 3]
```diff
@@ -35,21 +35,21 @@ AR21,,AR30-03,,GI02HYPM-WFP02-04-CTDPFL000,4,1,2017-08-06T13:45:00,2018-06-16T06
...
+AR30-03,,AR35-05,,GI02HYPM-WFP02-01-FLORDL000,5,1,2018-06-09T12:55:00,2019-08-10T06:34:00,CGMGI-02HYPM-00005,CGVEH-GWFPGX-10401,CGINS-FLORDL-03280,59.96974,-39.49005,,2573,2669,
+AR30-03,,AR35-05,,GI02HYPM-WFP02-03-DOSTAL000,5,1,2018-06-09T12:55:00,2019-08-10T06:34:00,CGMGI-02HYPM-00005,CGVEH-GWFPGX-10401,CGINS-DOSTAL-01476,59.96974,-39.49005,,2573,2669,
+AR30-03,,AR35-05,,GI02HYPM-WFP02-04-CTDPFL000,5,1,2018-06-09T12:55:00,2019-08-10T06:34:00,CGMGI-02HYPM-00005,CGVEH-GWFPGX-10401,CGINS-CTDPFL-00131,59.96974,-39.49005,,2573,2669,
+AR30-03,,AR35-05,,GI02HYPM-WFP02-05-VEL3DL000,5,1,2018-06-09T12:55:00,2019-08-10T06:34:00,CGMGI-02HYPM-00005,CGVEH-GWFPGX-10401,CGINS-VEL3DL-01109,59.96974,-39.49005,,2573,2669,
+AR30-03,,AR35-05,,GI02HYPM-MPM01-02-ZPLSGA009,5,1,2018-06-09T12:55:00,2019-08-10T06:34:00,CGMGI-02HYPM-00005,,CGINS-ZPLSGA-55109,59.96974,-39.49005,,,2669,upward-looking
+AR30-03,,AR35-05,,GI02HYPM-MPM01-02-ZPLSGA010,5,1,2018-06-09T12:55:00,2019-08-10T06:34:00,CGMGI-02HYPM-00005,,CGINS-ZPLSGA-55111,59.96974,-39.49005,,,2669,downward-looking
```
Comment (snwhite) [inline deployment/GI02HYPM_Deploy.csv]: For the recovered HYPM-5 at Location B...
Both ZPLSG instruments should have a deployment depth of 150 m.
WFP instruments should have a deployment depth of 2560 m.
The CTDMO should have a deployment depth of 160 m.
The SIOCON should have a deployment depth of 2570 m.

File: deployment/GI05MOAS-PG515_Deploy.csv [hunk 4]
```diff
@@ -1,8 +1,8 @@
...
+AR35-05,,AR35-05,,GI05MOAS-PG515-01-CTDGVM000,3,1,2019-08-06T17:20:00,2019-08-06T21:28:00,CGVGI-05MOAS-00000,CGVEH-GLDRGP-00515,CGINS-CTDGVM-09231,59.9162,-39.26579,,200,200,glider leaked
+AR35-05,,AR35-05,,GI05MOAS-PG515-02-DOSTAM000,3,1,2019-08-06T17:20:00,2019-08-06T21:28:00,CGVGI-05MOAS-00000,CGVEH-GLDRGP-00515,CGINS-DOSTAM-00414,59.9162,-39.26579,,200,200,glider leaked
+AR35-05,,AR35-05,,GI05MOAS-PG515-03-FLORTM000,3,1,2019-08-06T17:20:00,2019-08-06T21:28:00,CGVGI-05MOAS-00000,CGVEH-GLDRGP-00515,CGINS-FLORTM-03501,59.9162,-39.26579,,200,200,glider leaked
+AR35-05,,AR35-05,,GI05MOAS-PG515-04-FLORTO000,3,1,2019-08-06T17:20:00,2019-08-06T21:28:00,CGVGI-05MOAS-00000,CGVEH-GLDRGP-00515,CGINS-FLORTO-01239,59.9162,-39.26579,,200,200,glider leaked
+AR35-05,,AR35-05,,GI05MOAS-PG515-05-NUTNRM000,3,1,2019-08-06T17:20:00,2019-08-06T21:28:00,CGVGI-05MOAS-00000,CGVEH-GLDRGP-00515,CGINS-NUTNRM-00760,59.9162,-39.26579,,200,200,glider leaked
+AR35-05,,AR35-05,,GI05MOAS-PG515-06-PARADM000,3,1,2019-08-06T17:20:00,2019-08-06T21:28:00,CGVGI-05MOAS-00000,CGVEH-GLDRGP-00515,CGINS-PARADM-50185,59.9162,-39.26579,,200,200,glider leaked
```
Comment (snwhite) [inline deployment/GI05MOAS-PG515_Deploy.csv]: I'd like to get confirmation from @cdobs on what the note should say.  I believe it is an "aft glider leak"

File: deployment/GI03FLMB_Deploy.csv (hunk 1 above)
Comment (rtravis-whoi) [reply deployment/GI03FLMB_Deploy.csv]: This is what you sent via email:
Please note that the deployed depths of the OSNAP instruments on the 2 Flanking Moorings will need to be updated:
Off Bottom 	FLMA-6   	FLMB-6
   1000 m 	1692 m     	1819 m 
    700 m   	1992 m    	2119 m 
    400 m   	2292 m    	2419 m 
    100 m   	2592 m    	2719 m

File: deployment/GI02HYPM_Deploy.csv (hunk 2 above)
Comment (rtravis-whoi) [reply deployment/GI02HYPM_Deploy.csv]: Oh ok - your email said that all other depths should be correct.  Will fix.

File: deployment/GI02HYPM_Deploy.csv (hunk 3 above)
Comment (rtravis-whoi) [reply deployment/GI02HYPM_Deploy.csv]: Same as above - thought all other depths were correct.

File: deployment/GI05MOAS-PG515_Deploy.csv (hunk 4 above)
Comment (rtravis-whoi) [reply deployment/GI05MOAS-PG515_Deploy.csv]: Roger that - table just stated 'leak detect'.  Will wait for his input.

File: deployment/GI03FLMB_Deploy.csv (hunk 1 above)
Comment (snwhite) [reply deployment/GI03FLMB_Deploy.csv]: My note was for FLMB-6.  This is FLMB-5 which was just recovered.  If the deployment depth is 2829 m, then 1000 m off the bottom is 1829 m, not 1830 m which what is listed.

Comment (rtravis-whoi) [reply deployment/GI03FLMB_Deploy.csv]: ok then.  Fixing.

File: deployment/GI02HYPM_Deploy.csv (hunk 3 above)
Comment (rtravis-whoi) [reply deployment/GI02HYPM_Deploy.csv]: fixed

File: deployment/GI02HYPM_Deploy.csv (hunk 2 above)
Comment (rtravis-whoi) [reply deployment/GI02HYPM_Deploy.csv]: fixed

File: deployment/GI05MOAS-PG515_Deploy.csv (hunk 4 above)
Comment (snwhite) [reply deployment/GI05MOAS-PG515_Deploy.csv]: Collin says "glider leaked" is fine.

File: deployment/GI01SUMO_Deploy.csv [hunk 5]
```diff
@@ -201,103 +201,103 @@ AR21,,AR30-03,,GI01SUMO-SBD12-04-PCO2AA000,4,1,2017-08-05T18:17:00,2018-06-14T08
...
+AR35-05,,,,GI01SUMO-SBD11-08-NUTNRB000,6,1,2019-08-05T15:26:00,,CGMGI-01SUMO-00006,,CGINS-NUTNRB-01092,59.945118,-39.57401,,1,2664,
+AR35-05,,,,GI01SUMO-SBD12-00-DCLENG000,6,1,2019-08-05T15:26:00,,CGMGI-01SUMO-00006,,CGCON-BDCL12-50009,59.945118,-39.57401,,0,2664,
+AR35-05,,,,GI01SUMO-SBD12-01-OPTAAD000,6,1,2019-08-05T15:26:00,,CGMGI-01SUMO-00006,,CGINS-OPTAAD-00187,59.945118,-39.57401,,1,2664,
+AR35-05,,,,GI01SUMO-SBD12-02-FLORTD000,6,1,2019-08-05T15:26:00,,CGMGI-01SUMO-00006,,CGINS-FLORTD-01102,59.945118,-39.57401,,1,2664,
+AR35-05,,,,GI01SUMO-SBD12-03-HYDGN0000,6,1,2019-08-05T15:26:00,,CGMGI-01SUMO-00006,,CGINS-HYDGN0-49876,59.945118,-39.57401,,0,2664,
+AR35-05,,,,GI01SUMO-SBD12-04-PCO2AA000,6,1,2019-08-05T15:26:00,,CGMGI-01SUMO-00006,,CGINS-PCO2AA-35262,59.945118,-39.57401,,0,2664,
+AR35-05,,,,GI01SUMO-SBD12-05-WAVSSA000,6,1,2019-08-05T15:26:00,,CGMGI-01SUMO-00006,,CGINS-WAVSSA-06301,59.945118,-39.57401,,0,2664,
+AR35-05,,,,GI01SUMO-SBD12-06-METBKA000,6,1,2019-08-05T15:26:00,,CGMGI-01SUMO-00006,,CGINS-METLGR-00036,59.945118,-39.57401,,-5,2664,
+AR35-05,,,,GI01SUMO-SBD12-08-FDCHPA000,6,1,2019-08-05T15:26:00,,CGMGI-01SUMO-00006,,CGINS-FDCHPA-64403,59.945118,-39.57401,,-5,2664,
```
Comment (reedan88) [inline deployment/GI01SUMO_Deploy.csv]: Looks good.

File: deployment/GI02HYPM_Deploy.csv [hunk 6]
```diff
@@ -35,21 +35,21 @@ AR21,,AR30-03,,GI02HYPM-WFP02-04-CTDPFL000,4,1,2017-08-06T13:45:00,2018-06-16T06
...
+AR35-05,,,,GI02HYPM-RIM01-02-CTDMOG039,6,1,2019-08-06T12:37:00,,CGMGI-02HYPM-00006,,CGINS-CTDMOG-11611,59.97287,-39.53082,,160,2664,
+AR35-05,,,,GI02HYPM-WFP02-00-WFPENG000,6,1,2019-08-06T12:37:00,,CGMGI-02HYPM-00006,CGVEH-GWFPGX-77401,CGCON-GWFPGX-77401,59.97287,-39.53082,,2560,2664,
+AR35-05,,,,GI02HYPM-WFP02-01-FLORDL000,6,1,2019-08-06T12:37:00,,CGMGI-02HYPM-00006,CGVEH-GWFPGX-77401,CGINS-FLORDL-02350,59.97287,-39.53082,,2560,2664,
+AR35-05,,,,GI02HYPM-WFP02-03-DOSTAL000,6,1,2019-08-06T12:37:00,,CGMGI-02HYPM-00006,CGVEH-GWFPGX-77401,CGINS-DOSTAL-01106,59.97287,-39.53082,,2560,2664,
+AR35-05,,,,GI02HYPM-WFP02-04-CTDPFL000,6,1,2019-08-06T12:37:00,,CGMGI-02HYPM-00006,CGVEH-GWFPGX-77401,CGINS-CTDPFL-00097,59.97287,-39.53082,,2560,2664,
+AR35-05,,,,GI02HYPM-WFP02-05-VEL3DL000,6,1,2019-08-06T12:37:00,,CGMGI-02HYPM-00006,CGVEH-GWFPGX-77401,CGINS-VEL3DL-01025,59.97287,-39.53082,,2560,2664,
+AR35-05,,,,GI02HYPM-MPM01-02-ZPLSGA009,6,1,2019-08-06T12:37:00,,CGMGI-02HYPM-00006,,CGINS-ZPLSGA-55069,59.97287,-39.53082,,150,2664,upward-looking
+AR35-05,,,,GI02HYPM-MPM01-02-ZPLSGA010,6,1,2019-08-06T12:37:00,,CGMGI-02HYPM-00006,,CGINS-ZPLSGA-55066,59.97287,-39.53082,,150,2664,downward-looking
```
Comment (reedan88) [inline deployment/GI02HYPM_Deploy.csv]: Looks good.

File: deployment/GI03FLMA_Deploy.csv [hunk 7]
```diff
@@ -103,55 +103,55 @@ AR21,,AR30-03,,GI03FLMA-RIS01-00-SIOENG000,4,1,2017-08-08T15:59:00,2018-06-15T07
...
+AR35-05,,,,GI03FLMA-RIM01-02-CTDMOG048,6,1,2019-08-07T13:34:00,,CGMGI-03FLMA-00006,,CGINS-CTDMOG-10256,59.77010,-39.88419,,500,2692,
+AR35-05,,,,GI03FLMA-RIM01-02-CTDMOH049,6,1,2019-08-07T13:34:00,,CGMGI-03FLMA-00006,,CGINS-CTDMOH-10265,59.77010,-39.88419,,750,2692,
+AR35-05,,,,GI03FLMA-RIM01-02-CTDMOH050,6,1,2019-08-07T13:34:00,,CGMGI-03FLMA-00006,,CGINS-CTDMOH-10227,59.77010,-39.88419,,1000,2692,
+AR35-05,,,,GI03FLMA-RIM01-02-CTDMOH051,6,1,2019-08-07T13:34:00,,CGMGI-03FLMA-00006,,CGINS-CTDMOH-11597,59.77010,-39.88419,,1500,2692,
+AR35-05,,,,GI03FLMA-RIS01-00-SIOENG000,6,1,2019-08-07T13:34:00,,CGMGI-03FLMA-00006,,CGCON-SIOCON-13992,59.77010,-39.88419,,30,2692,
+AR35-05,,,,GI03FLMA-RIS01-03-DOSTAD000,6,1,2019-08-07T13:34:00,,CGMGI-03FLMA-00006,,CGINS-DOSTAD-00515,59.77010,-39.88419,,30,2692,
+AR35-05,,,,GI03FLMA-RIS01-04-PHSENF000,6,1,2019-08-07T13:34:00,,CGMGI-03FLMA-00006,,CGINS-PHSENF-P0141,59.77010,-39.88419,,30,2692,
+AR35-05,,,,GI03FLMA-RIS01-05-FLORTD000,6,1,2019-08-07T13:34:00,,CGMGI-03FLMA-00006,,CGINS-FLORTD-01212,59.77010,-39.88419,,30,2692,
```
Comment (reedan88) [inline deployment/GI03FLMA_Deploy.csv]: Looks good.

File: deployment/GI03FLMB_Deploy.csv [hunk 8]
```diff
@@ -103,55 +103,55 @@ AR21,,AR30-03,,GI03FLMB-RIS01-00-SIOENG000,4,1,2017-08-07T15:49:00,2018-06-11T19
...
+AR35-05,,,,GI03FLMB-RIM01-02-CTDMOG068,6,1,2019-08-08T13:20:00,,CGMGI-03FLMB-00006,,CGINS-CTDMOG-12190,59.720515,-39.35474,,500,2819,
+AR35-05,,,,GI03FLMB-RIM01-02-CTDMOH069,6,1,2019-08-08T13:20:00,,CGMGI-03FLMB-00006,,CGINS-CTDMOH-12227,59.720515,-39.35474,,750,2819,
+AR35-05,,,,GI03FLMB-RIM01-02-CTDMOH070,6,1,2019-08-08T13:20:00,,CGMGI-03FLMB-00006,,CGINS-CTDMOH-11703,59.720515,-39.35474,,1000,2819,
+AR35-05,,,,GI03FLMB-RIM01-02-CTDMOH071,6,1,2019-08-08T13:20:00,,CGMGI-03FLMB-00006,,CGINS-CTDMOH-12219,59.720515,-39.35474,,1500,2819,
+AR35-05,,,,GI03FLMB-RIS01-00-SIOENG000,6,1,2019-08-08T13:20:00,,CGMGI-03FLMB-00006,,CGCON-SIOCON-12372,59.720515,-39.35474,,30,2819,
+AR35-05,,,,GI03FLMB-RIS01-03-DOSTAD000,6,1,2019-08-08T13:20:00,,CGMGI-03FLMB-00006,,CGINS-DOSTAD-00224,59.720515,-39.35474,,30,2819,
+AR35-05,,,,GI03FLMB-RIS01-04-PHSENF000,6,1,2019-08-08T13:20:00,,CGMGI-03FLMB-00006,,CGINS-PHSENF-P0183,59.720515,-39.35474,,30,2819,
+AR35-05,,,,GI03FLMB-RIS01-05-FLORTD000,6,1,2019-08-08T13:20:00,,CGMGI-03FLMB-00006,,CGINS-FLORTD-01221,59.720515,-39.35474,,30,2819,
```
Comment (reedan88) [inline deployment/GI03FLMB_Deploy.csv]: Looks good!

File: deployment/GI05MOAS-PG515_Deploy.csv (hunk 4 above)
Comment (reedan88) [reply deployment/GI05MOAS-PG515_Deploy.csv]: Looks good.

File: deployment/GI05MOAS-GL525_Deploy.csv [hunk 9]
```diff
@@ -1,5 +1,5 @@
...
-AR35-05,,,,GI05MOAS-GL525-00-ENG000000,2,1,,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00525,CGCON-GLDROO-00525,,,,1000,1000,
-AR35-05,,,,GI05MOAS-GL525-01-FLORDM000,2,1,,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00525,CGINS-FLORDM-03579,,,,1000,1000,
-AR35-05,,,,GI05MOAS-GL525-02-DOSTAM000,2,1,,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00525,CGINS-DOSTAM-00418,,,,1000,1000,
-AR35-05,,,,GI05MOAS-GL525-04-CTDGVM000,2,1,,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00525,CGINS-CTDGVM-09232,,,,1000,1000,
+AR35-05,,,,GI05MOAS-GL525-00-ENG000000,2,1,2019-08-06T17:23:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00525,CGCON-GLDROO-00525,59.91594,-39.26567,,1000,1000,
+AR35-05,,,,GI05MOAS-GL525-01-FLORDM000,2,1,2019-08-06T17:23:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00525,CGINS-FLORDM-03579,59.91594,-39.26567,,1000,1000,
+AR35-05,,,,GI05MOAS-GL525-02-DOSTAM000,2,1,2019-08-06T17:23:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00525,CGINS-DOSTAM-00418,59.91594,-39.26567,,1000,1000,
+AR35-05,,,,GI05MOAS-GL525-04-CTDGVM000,2,1,2019-08-06T17:23:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00525,CGINS-CTDGVM-09232,59.91594,-39.26567,,1000,1000,
```
Comment (reedan88) [inline deployment/GI05MOAS-GL525_Deploy.csv]: Looks good.

File: deployment/GI05MOAS-GL560_Deploy.csv [hunk 10]
```diff
@@ -1,5 +1,5 @@
...
-AR35-05,,,,GI05MOAS-GL560-00-ENG000000,2,1,,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00560,CGCON-GLDROO-00560,,,,1000,1000,
-AR35-05,,,,GI05MOAS-GL560-01-FLORDM000,2,1,,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00560,CGINS-FLORDM-04095,,,,1000,1000,
-AR35-05,,,,GI05MOAS-GL560-02-DOSTAM000,2,1,,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00560,CGINS-DOSTAM-00437,,,,1000,1000,
-AR35-05,,,,GI05MOAS-GL560-04-CTDGVM000,2,1,,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00560,CGINS-CTDGVM-09268,,,,1000,1000,
+AR35-05,,,,GI05MOAS-GL560-00-ENG000000,2,1,2019-08-06T18:05:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00560,CGCON-GLDROO-00560,59.91381,-39.26488,,1000,1000,
+AR35-05,,,,GI05MOAS-GL560-01-FLORDM000,2,1,2019-08-06T18:05:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00560,CGINS-FLORDM-04095,59.91381,-39.26488,,1000,1000,
+AR35-05,,,,GI05MOAS-GL560-02-DOSTAM000,2,1,2019-08-06T18:05:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00560,CGINS-DOSTAM-00437,59.91381,-39.26488,,1000,1000,
+AR35-05,,,,GI05MOAS-GL560-04-CTDGVM000,2,1,2019-08-06T18:05:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00560,CGINS-CTDGVM-09268,59.91381,-39.26488,,1000,1000,
```
Comment (reedan88) [inline deployment/GI05MOAS-GL560_Deploy.csv]: Looks good.

**Pull request merged**